src/NuGet.config

```diff
@@ -3,5 +3,6 @@
   <packageSources>
     <clear />
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
+    <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
```

Comment:
This was needed to build locally

Reply:
I'm not sure if it should be. Do you remember what dependency failed to resolve without this package source?

Reply:
Yes, this one:
`    <PackageReference Condition="'$(TargetGroup)'=='netcoreapp'" Include="Microsoft.DotNet.RemoteExecutor" Version="$(MicrosoftDotnetRemoteExecutorVersion)" /> 
`

Reply:
Since you have the cached version of the package source, can you remove this please? There will be critical error message on the pipelines if this gets merged.

Reply:
Done